doc/user-guide/io.rst

```diff
@@ -153,83 +153,31 @@ to the original netCDF file, regardless if they exist in the original dataset.
 Groups
 ~~~~~~
 
-NetCDF groups are not supported as part of the :py:class:`Dataset` data model.
-Instead, groups can be loaded individually as Dataset objects.
-To do so, pass a ``group`` keyword argument to the
-:py:func:`open_dataset` function. The group can be specified as a path-like
-string, e.g., to access subgroup ``'bar'`` within group ``'foo'`` pass
-``'/foo/bar'`` as the ``group`` argument.
-
-In a similar way, the ``group`` keyword argument can be given to the
-:py:meth:`Dataset.to_netcdf` method to write to a group
-in a netCDF file.
-When writing multiple groups in one file, pass ``mode='a'`` to
-:py:meth:`Dataset.to_netcdf` to ensure that each call does not delete the file.
-For example:
-
-.. ipython::
-    :verbatim:
-
-    In [1]: ds1 = xr.Dataset({"a": 0})
-
-    In [2]: ds2 = xr.Dataset({"b": 1})
-
-    In [3]: ds1.to_netcdf("file.nc", group="A")
-
-    In [4]: ds2.to_netcdf("file.nc", group="B", mode="a")
-
-We can verify that two groups have been saved using the ncdump command-line utility.
-
-.. code:: bash
-
-    $ ncdump file.nc
-    netcdf file {
-
-    group: A {
-      variables:
-        int64 a ;
-      data:
-
-       a = 0 ;
-      } // group A
-
-    group: B {
-      variables:
-        int64 b ;
-      data:
-
-       b = 1 ;
-      } // group B
-    }
-
-Either of these groups can be loaded from the file as an independent :py:class:`Dataset` object:
-
-.. ipython::
-    :verbatim:
-
-    In [1]: group1 = xr.open_dataset("file.nc", group="A")
+Whilst netCDF groups can only be loaded individually as ``Dataset`` objects, a
+whole file of many nested groups can be loaded as a single
+:py:class:`xarray.DataTree` object. To open a whole netCDF file as a tree of groups
+use the :py:func:`xarray.open_datatree` function. To save a DataTree object as a
+netCDF file containing many groups, use the :py:meth:`xarray.DataTree.to_netcdf` method.
```

Comment:
I'm not sure which other instructions you mean here.

Reply:
If you look at the diff, there's a long set of how to do grouped netcdf files without DataTrees.  I'm guessing it's OBE, but just checking.

Reply:
I finally got back to this thread (sorry!)

I'm still a little unclear. Do you mean the lines that were removed from this file (L161-L231 prior to this PR)?

I skimmed quickly through for all references to "group" in the documentation, and added a tip slightly further up in `doc/user-guide/io.rst`, where it talks about using `a` as the write mode if writing multiple groups to avoid clobbering things. Maybe that covers things?

Reply:
Yes, those lines are what I was talking about. I think we're covered because we don't want to do it that way anymore. We just want people to use `DataTree.to_netcdf()`